bot.py

```
@@ -6,16 +6,16 @@
 import re
 import os
 import json
-from typing import Optional
+from typing import Optional, Dict, Any, Union
 from datetime import datetime, timedelta, timezone
```

Flake8 (7.2.0)

[error] 'datetime.datetime' imported but unused

(F401)